pyoctoprintapi/octoprint_client.py

```diff
@@ -98,4 +100,34 @@ async def pause_job(self) -> None:
 
     async def resume_job(self) -> None:
         _LOGGER.debug("Sending resume job command")
-        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
\ No newline at end of file
+        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
+
+    async def disconnect(self) -> None:
+        _LOGGER.debug("Disconnecting from printer")
+        await self._api.issue_connection_command("disconnect")
+
+    async def connect(self, port: Optional[str] = None, baud_rate: int = None, printer_profile: str = None, save: bool = None, auto_connect: bool = None) -> None:
+        _LOGGER.debug("Connecting to printer")
+        await self._api.issue_connection_command("connect", port=port, baudrate=baud_rate, printerprofile=printer_profile, save=save, autoconnect=auto_connect)
```

Comment:
Can the command string be moved into the consts.py file?
```suggestion
        await self._api.issue_connection_command(CONNECTION_COMMAND_CONNECT, port=port, baudrate=baud_rate, printerprofile=printer_profile, save=save, autoconnect=auto_connect)
```